ikarus/utils/traits.hh

```diff
@@ -110,13 +114,67 @@ struct hasType<T, std::tuple<T, Ts...>> : std::true_type
 };
 #endif
 
+/**
+ * \brief Function to return the n-th object of a variadic template.
+ *
+ * \ingroup traits
+ *
+ */
+struct NthArgument
```

Comment:
I didn't find a place where this `struct` was used and its purpose of addition then is not clear to me. May be this was used in Python bindings, I am not sure. Is it just an add functionality, if in case needed?

Reply:
You are right this is not needed anymore